docs/installation/docker.md

```diff
@@ -118,12 +118,14 @@ BEE_SWAP_ENDPOINT=https://stake.getblock.io/mainnet/?api_key=your-api-key
 BEE_PASSWORD=my-password
 ```
 
-To enable Clef support, we must also change the following params: 
+To enable Clef support on mainnet, we must also change the following params: 
 
 ```
-CLEF_CHAINID=5
+BEE_CLEF_CHAIN_ID=100
```

Comment:
i think it is just `CLEF_CHAINID` for docker. please verify.

Reply:
verified!